src/components/dashboard/fleet/FleetDashboard.jsx

```
@@ -1,7 +1,8 @@
 import React, { useState, useEffect, useCallback, useMemo, useRef } from 'react';
 import {
   Search, Filter, Download,
-  RefreshCw, Map, Ship, AlertTriangle
+  RefreshCw, Map, Ship, AlertTriangle,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)